internal/guest/runtime/hcsv2/spec_devices.go

```diff
@@ -83,15 +88,19 @@ func devicePathFromPCIPath(pciPath string) (*devices.Device, error) {
 		syfsDevPath := fmt.Sprintf(sysfsDevPathFormat, deviceTypeString, major, minor)
 		sysfsFullPath, err := filepath.EvalSymlinks(syfsDevPath)
 		if err != nil {
-			return nil, err
+			// Some drivers will make dev nodes that do not have a matching block or
+			// char device -- skip those.
+			continue
```

Comment:
should we log the errors here?

Reply:
Would you prefer it be debug logs or warn logs? My only concern is if there are a lot of logs that get printed every time a container tries to add an assigned device in a pod.

Reply:
debug is probably best, if its gonna be a lot of them